src/plugins/intel_gpu/src/graph/impls/onednn/gemm_onednn.cpp

```diff
@@ -301,6 +301,14 @@ struct gemm_onednn : typed_primitive_onednn_impl<gemm> {
     }
 
     static std::unique_ptr<primitive_impl> create(const gemm_node& arg, const kernel_impl_params& impl_params) {
+        bool full_tensor_or_per_tensor = true;
+        for (auto prim : arg.get_fused_primitives()) {
+            full_tensor_or_per_tensor &=
+                prim.input_layout.count() == prim.output_layout.count() || prim.input_layout.count() == 1;
+        }
+        if (!full_tensor_or_per_tensor) {
+            IE_THROW() << "FIXME: Onednn gemm has some issues in the post-op axis. Refer PR(#15353) message.";
```

Comment:
Please state clearly what is not supported. Better message would be "Unimplemented: Onednn gemm does not support per channel binary post-operation"

Reply:
Done.